lib/ansible/modules/notification/catapult.py

```diff
@@ -0,0 +1,165 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# (c) 2016, Jonathan Mainguy <jon@soh.re>
+#
+# This file is part of Ansible
+#
+# Ansible is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# Ansible is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with Ansible.  If not, see <http://www.gnu.org/licenses/>.
+
+DOCUMENTATION = '''
+---
+module: catapult
+version_added: 2.4
+short_description: Send a sms / mms using the catapult bandwidth api
+description:
+     - The M(catapult) module allows notifications to be sent using sms / mms via the catapult bandwidth api.
```

Comment:
`Allows notifications to be sent using SMS or MMS via the catapult bandwidth API.`

No need to repeat the module name,